src/clj/rems/css/styles.clj

```diff
@@ -473,6 +473,7 @@
    [:.alert-dark {:color (get-theme-attribute :alert-dark-color)
                   :background-color (get-theme-attribute :alert-dark-bgcolor)
                   :border-color (get-theme-attribute :alert-dark-bordercolor :alert-dark-color)}]
+   [:.applications.alert-secondary {:margin-top "1rem"}]
```

Comment:
How about instead of this, we just add `mt-3` to this line https://github.com/CSCfi/rems/blob/master/src/cljs/rems/application_list.cljs#L181

The we would know with certainty that it will ever only affect that one place that needed fixing.

Reply:
I think this line can just be removed now?

Reply:
This line is what fixes the problem, why remove it?